oauthenticator/openshift.py

```diff
@@ -153,7 +156,14 @@ def token_request(url):
             headers=headers,
         )
 
-        ocp_user = await self.fetch(req)
+        ocp_user = {}
+        try:
+            ocp_user = await self.fetch(req) #TODO: tornado.httpclient.HTTPClientError: HTTP 401: Unauthorized
+        except HTTPError as ex:
+            if ex.code == 401:
+                return None
```

Comment:
Is 401 the only HTTP we need to handle, what about 403: Forbidden?

Reply:
This call is to get user's own data from OpenShift OAuth API, so it cannot return Forbidden - it can only return the data or 401...or potentially any other error, but that results in exception which is expected